doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc

```diff
@@ -107,6 +107,10 @@ For a complete list of supported dialects, see the _Hibernate SQL Dialects_ tabl
 .. *Host*: Enter the host name of the external database server.
 .. *Port*: Enter the port number of the external database server.
 .. *Jdbc URL*: Enter the JDBC URL for the external database server.
+[NOTE]
+====
+If you are using the EntrepriseDB Postgres database server, do not use an URL starting with `jdbc:edb://`. Instead, use an URL starting with `jdbc:postgresql://`.
```

Comment:
Minor suggestion - rephrase as a positive instruction and refer to text field. This may improve translate-ability:

s/If you are using the EntrepriseDB Postgres database server, do not use an URL starting with `jdbc:edb://`. Instead, use an URL starting with `jdbc:postgresql://`. / If you use an EnterpriseDB Prosgres database server, ensure the `Jdbc URL` field URL starts with `jdbc:postgresql://` rather than `jdbc:edb://`.